manifests/09-prometheus-rules.yaml

```
@@ -17,3 +17,15 @@ spec:
         message: |
           Image Registry Storage configuration has changed in the last 30
           minutes. This change may have caused data loss.
+    - alert: ImportImageUsingV1Protocol
+      expr: increase(apiserver_v1_image_imports[15m]) > 0
+      labels:
+        severity: warning
+      annotations:
+        message: |
+          An attempt to import an image using deprecated registry protocol v1
+          was detected on pod {{ $labels.pod }}. Support for protocol v1 will
+          be deprecated, it is recommended to use an image registry who supports
+          a newer and safest protocol. You can see what image got imported using
```

```suggestion
          the newer v2 protocol. You can see what image was imported using
```

i'd rather not get into judgement type evaluations here.